src/components/Import/DescriptionImport.vue

```diff
@@ -3,6 +3,7 @@
     <div v-if="isLoading">
       <loading-modal /><br><br>
     </div>
+    <div class="notification fail" v-if="allUnitTypesZero">Notice: The unit type quantity is currently 0.00. Upon saving, the description will be displayed as 'Head'.</div>
```

Comment:
Suggest to change to "Warning: One or more unit type quantities are currently set to 0. Please check before importing the BQ items."